.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,58 @@
+name: Bug Report (Beta)
+about: Report a Bug or an Issue with Slimefun 4.
+title: ''
+labels: ['\U0001F3AF Needs testing', '\U0001F41E Bug Report']
+assignees: ''
+body:
+- type: markdown
+  attributes:
+    value: '## Thanks for reporting a bug!'
+- type: textarea
+  label: ':round_pushpin: Description'
+  validations:
+    required: true
+  description: |
+    A clear and detailed description of what went wrong.
+    The more information you can provide, the easier we can handle this problem.
+- type: textarea
+  label: ':bookmark_tabs: Reproduction Steps'
+  validations:
+    required: true
+  description: |
+    Tell us the exact steps to reproduce this issue, the more detailed the easier we can reproduce it.
+    Youtube Videos and Screenshots are recommended!
+  value: |
+    1.
+    2.
+    3.
+- type: textarea
+  label: ':bulb: Expected Behavior'
+  validations:
+    required: true
+  description: |
+    What were you expecting to happen?
+    What do you think would have been the correct behaviour?
+- type: textarea
+  label: ':scroll: Server Log'
+  description: |
+    Take a look at your Server Log and post any errors you can find via https://pastebin.com/
+    If you are unsure about it, post your full log, you can find it under /logs/latest.log
+- type: textarea
+  label: ':open_file_folder: /error-reports/ Folder'
+  description: |
+    Check the folder /plugins/Slimefun/error-reports/ and upload any files inside that folder.
+    You can also post these files via https://pastebin.com/
+- type: textarea
+  label: ':compass: Environment'
+  validations:
+    required: true
+  description: |
+    Any issue without the exact version numbers will be closed!
+    "latest" IS NOT A VERSION NUMBER. -->
+    We recommend running "/sf versions" and showing us a screenshot of that.
+    Make sure that the screenshot covers the entire output of that command.
+    If your issue is related to other plugins, make sure to include the versions of these plugins too!
+  value: |
+    - Server software:
```

Comment:
I think we should use a dropdown for this. Mainly we are interested in whether they use Spigot, Paper or some other fork.

I would suggest changing the Server Software to a dedicated dropdown with the following options:
* Spigot
* Paper
* Tuinity
* Yatopia
* Purpur
* Airplane
* Other

(or just Spigot, Paper and other.)

Reply:
Yeah, the only thing is how much do we care about the version of it.

Reply:
Ultimately I would still prefer a screenshot of `/sf versions` I guess

Reply:
I wish we could accept only a file/link. I may have to send that as a suggestion

Reply:
Also is there any way the `validations` field can be used for... proper input validation? :eyes:

Reply:
One nice thing is that we can accept input which is automatically highlighted. So for the logs we can make the text there highlighted with like java or something

Reply:
what?

Reply:
So, we have an input for log file and if they decide to send text they can and it'll turn into a code block

"Post your log file"
```java
enabling slimefun
exception NullPointerException
  at some.class.Slimefun:10
```

would be nice if we ever wanted configs too, we could highlight with YAML